mne/epochs.py

```diff
@@ -402,6 +409,8 @@ def decimate(self, decim, copy=False):
         else:
             epochs._decim_slice = decim_slice
             epochs.times = epochs._raw_times[epochs._decim_slice]
+        epochs.tmin = epochs.times[0]
+        epochs.tmax = epochs.times[-1]
```

Comment:
Decimation could cause the last sample, which first corrsponded to 1.1s, to now correspond to 1.0s instead. By updating tmin/tmax to the actual time range represented by the samples in your data you would remove some confusion I think.